monai/transforms/signal/radial_fourier.py

```
@@ -0,0 +1,359 @@
+            img: input medical image data. Expected shape: (..., D, H, W)
+                 where D, H, W are spatial dimensions.
+
+        Returns:
+            Transformed data in radial frequency domain.
+        """
+        # Convert to tensor if needed
+        img_tensor, *_ = convert_data_type(img, torch.Tensor)
+        # Get spatial dimensions
+        spatial_shape = tuple(img_tensor.shape[d] for d in self.spatial_dims)
+        if len(spatial_shape) != 3:
+            raise ValueError(f"Expected 3 spatial dimensions, got {len(spatial_shape)}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)